charts/datadog/templates/_helpers.tpl

```diff
@@ -574,6 +586,14 @@ datadog-agent-fips-config
 {{- end -}}
 {{- end -}}
 
+{{- define "agents-install-otel-configmap-name" -}}
+{{- if .Values.providers.gke.autopilot -}}
+datadog-agent-otel-config
+{{- else -}}
```

Comment:
I think this should be removed since the OTEL container hasn't been added to the GKE Autopilot datadog allowlist

Reply:
updated.